### PR TITLE
Compress and About Pages added

### PR DESCRIPTION
**Pull request by heliumonoxide**
compress and about pages added but there are still needs for adding throw error when the file upload doesn't work as in the commented section of compress page. also, haven't add filedialog yet to the compress page

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -29,7 +31,8 @@ def show_frame(self, page_name):
 
 
 if __name__ == "__main__":
-    app = SampleApp()
+    app = App()
     app.title("Video Compressor Application")
     app.geometry("950x560")
+    app.resizable(False, False)
```
Comment (vincadrn) [inline main.py:37]: Is this intentional, design-wise? What happens if the user try to resize the window?

File: pages/CompressPage.py [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+        self.image_image_9 = PhotoImage(
+            file=relative_to_assets("image_9.png"))
+        self.image_9 = canvas.create_image(
+            583.800048828125,
+            388.0,
+            image=self.image_image_9
+        )
+
+        # self.image_image_10 = PhotoImage(
```
Comment (vincadrn) [inline pages/CompressPage.py:234]: Could you please remove the comments if not used anymore?

**Review by vincadrn — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (heliumonoxide) [reply main.py:37]: yes, i made it into fixed-size window so that it won't be a problem with GUI later future

File: pages/CompressPage.py (hunk 2 above)
Comment (heliumonoxide) [reply pages/CompressPage.py:234]: it's still in use for throw error message on the GUI

**Pull request merged**